components/email/src/Email.svelte

```diff
@@ -387,7 +387,11 @@
   }
 
   async function handleThread(event: MouseEvent | KeyboardEvent) {
-    if (activeThread.messages.length <= 0) {
+    const messageType = isThreadADraftEmail(activeThread)
+      ? "drafts"
+      : "messages";
+
+    if (activeThread[messageType].length <= 0) {
```

Comment:
I wrote the comment above and then I saw this.... hmmmmm

Reply:
Alright, let's make an enum that tracks this so we don't have to deal with strings when checking the message type

Reply:
@ozsivanov Would you mind linking an example of this in the code base?

Reply:
Yep, something like https://github.com/nylas/components/blob/main/commons/src/enums/Availability.ts.

In the availability component, it gets used like this https://github.com/nylas/components/blob/main/components/availability/src/Availability.svelte#L1532

Reply:
@ozsivanov https://github.com/nylas/components/pull/316/commits/d08120a4339aeaaa373f2ebd45bd31acbe13edc2